examples/calculator_fractional/src/main.rs

```diff
@@ -44,7 +44,7 @@ struct Expression {
 enum ParseResult {
     Help,
     Exit,
-    Expression(Expression),
+    Expression(Box<Expression>),
```

Comment:
It's pretty cool that Clippy caught this, but for examples we need to value readability over all else. Box is a Rustism we don't want to bog users down with while getting ramped up. Add an ignore for this one.